src/libaktualizr/primary/sotauptaneclient.cc

```diff
@@ -1134,6 +1129,17 @@ void SotaUptaneClient::campaignAccept(const std::string &campaign_id) {
   report_queue->enqueue(std_::make_unique<CampaignAcceptedReport>(campaign_id));
 }
 
+bool SotaUptaneClient::isInstallCompletionRequired() {
+  bool force_install_completion = (hasPendingUpdates() and config.uptane.force_install_completion);
```

Comment:
We usually avoid these synonym boolean operators as they are quite uncommon in C and C++ contexts. Would prefer `&&` here.

Reply:
Yeah, I've done too much python coding recently :)